server/bleep/src/remotes/poll.rs

```diff
@@ -26,7 +26,7 @@ const POLL_INTERVAL_MINUTE: &[Duration] = &[
 
 pub(crate) async fn check_credentials(app: Application) {
     loop {
-        if app.env.use_aaa() {
+        if let Environment::PrivateServer = app.env {
```

Comment:
Don't do direct matching of these. Check for individual permissions bound to the server mode, as it was originally, this helps us understand the different modes the app can run in easily.